kernel/classes/ezscript.php

```diff
@@ -235,8 +235,17 @@ function startup()
     */
     function initialize()
     {
-        if( ob_get_length() != 0 )
-            ob_end_clean();
+        try
+        {
+            ezpKernel::instance();
```

Comment:
OK, I think the best would be to move `hasInstance()` method from [`\eZ\Publish\Core\MVC\Legacy\Kernel`](https://github.com/ezsystems/LegacyBridge/blob/master/mvc/Kernel.php#L71-L79) to `ezpKernel`. Then, instead of try/catch, you would do:

```php

```if ( !ezpKernel::hasInstance() )
{
    if ( ob_get_length() != 0 )
        ob_end_clean();
}